runtime_config_linux.go

```diff
@@ -198,6 +198,12 @@ type Network struct {
 	Priorities []InterfacePriority `json:"priorities"`
 }
 
+// IntelRdt for Linux cgroup 'intel_rdt' resource management
+type IntelRdt struct {
+	// L3 cache capacity bitmask (CBM) for container
+	L3Cbm *string `json:"l3Cbm,omitempty"`
```

Comment:
@vishh @wking Thank you for pointing this out.
In this cgroup context, intel_rdt.l3_cbm represents the L3 cache capacity **bitmask**. In kernel, by nature, read/write with 'hexadecimal string' is the best choice for user-friendly and user-readable. Kernel handles it as 'u64' via 'write_u64' interface inside for 'machine readability'.

In oci/specs, currently I choose the type 'string' instead of 'uint64' for some reasons:
1. JSON doesn't natively support hexadecimal, as @wking mentioned. For me it is really an important reason. If JSON supports hexadecimal or something more readable for bitmask, 'uint64' type inside with '0x' prefix is perfect, we don't need discuss this issue here any more.  
2. User-readability VS machine-readability. JSON doesn't support hexadecimal means user-readability issue even though kernel treats hexadecimal (0x-prefix), octal (0-prefix) and decimal integers without discrimination. For example (l3Cbm: 0xff0, bits 4-11 are set):
(1) Type string:
```json
"intel_rdt": {
    "l3Cbm": "0xff0"
}
```
(2) Type uint64:
```json
"intel_rdt": {
    "l3Cbm": 4080
}
```
(3) If JSON supports hexadecimal, with type uint64 looks like:
```json
"intel_rdt": {
    "l3Cbm": 0xff0
}
```
Case (3) should be perfect, but it doesn't work. Case (2), '4080' is very confusing as a bitmask value for end user. As a compromise, we use case (1) now.

Anyway, I will not always stick to my point if we have a better solution. Actually, when I was implementing https://github.com/opencontainers/runc/issues/433, I found the string/integer conversion back-and-forth is really chatty due to JSON's limitation on non-decimal integer support.